pdr_backend/cli/cli_arguments.py

```diff
@@ -112,6 +115,43 @@ def add_argument_LOOKBACK(self):
         )
 
 
+@enforce_types
+class NUM_Mixin:
+    def add_argument_NUM(self):
+        self.add_argument("NUM", type=int, help="1|2|..")
+
+
+@enforce_types
+class ACCOUNTS_Mixin:
+    def add_argument_ACCOUNTS(self):
+        self.add_argument(
+            "ACCOUNTS",
+            type=str,
```

Comment:
Similar to the stackoverflow I pasted for int types, please create a custom "csv_accounts" type that validates the account string. You can also find inspiration in dftool, where the functions are try/except blocks. The point is: by the time these arguments reach the command itself, they should be correct. It helps with UX a lot to get the error from the help/argparse part instead of looking for the failure inside the command